app/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ animealPlugin {
 }
 
 android {
+    namespace = "com.epmedu.animeal"
```

Comment:
`namespace` should be in all modules and need to remove `package` from AndroidManifest